src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -69,17 +64,16 @@ public void run() {
       if (photonPoseEstimator != null && photonCamera != null) {
         var photonResults = photonCamera.getLatestResult(); //Gets the latest camera results
 
-        // var test = VisionConstants.class.getDeclaredField("ROBOT_TO_" + photonCamera.getName().toUpperCase());
+        // var test = VisionConstants.class.getDeclaredField("ROBOT_TO_" + cameraName.toUpperCase());
         // test.setAccessible(true);
         // Transform3d test2 = (Transform3d) test.get(VisionConstants.class.getClass());
         //SmartDashboard.putString("Test", test2.toString());
```

Comment:
Is there a reason these are here but commented? Remove if no reason

Reply:
Will remove. This was some code from @aayushext , not sure if we planned to use it.